pkg/ansible/watches/watches.go

```diff
@@ -74,18 +76,49 @@ var (
 	manageStatusDefault                = true
 	watchDependentResourcesDefault     = true
 	watchClusterScopedResourcesDefault = false
+	selectorDefault                    = metav1.LabelSelector{}
 
 	// these are overridden by cmdline flags
 	maxWorkersDefault       = 1
 	ansibleVerbosityDefault = 2
 )
 
+// Creates, populates and returns Label Selector object during UnmarshalYAML
+func parseLabelSelector(dls tempLabelSelector) metav1.LabelSelector {
+	obj := metav1.LabelSelector{}
+	obj.MatchLabels = dls.MatchLabels
+
+	for _, v := range dls.MatchExpressions {
+		requirement := metav1.LabelSelectorRequirement{}
+		requirement.Key = v.Key
+		requirement.Operator = v.Operator
+		requirement.Values = v.Values
```

Comment:
Any specific reason you chose to initialize the variable like this? Normally I do it as one statement:

```golang
requirement := metav1.LabelSelectorRequirement{
    Key: v.Key,
    Operator: v.Operator,
    Values: v.Values,
}
```